src/index.js

```diff
@@ -47,32 +47,47 @@ const belowThreshold = (id, expected, categories) => {
   return actual < expected;
 };
 
-const getError = (id, expected, results) => {
-  const category = results.find((c) => c.id === id);
-  return `Expected category ${chalk.magenta(
+const getError = (id, expected, categories, audits) => {
+  const category = categories.find((c) => c.id === id);
+
+  const categoryError = `Expected category ${chalk.cyan(
     category.title,
   )} to be greater or equal to ${chalk.green(expected)} but got ${chalk.red(
     category.score !== null ? category.score : 'unknown',
   )}`;
+
+  const categoryAudits = category.auditRefs
+    .filter(({ weight, id }) => weight > 0 && audits[id].score < 1)
```

Comment:
Only list audits that donated to the category score based on weight, and also ignore audits with a perfect score